assets/airports/kchs.json

```diff
@@ -0,0 +1,10037 @@
+{
+    "airac": 2213,
+    "radio": {
+        "twr": "Charleston Tower",
+        "app": "Charleston Approach",
+        "dep": "Charleston Departure"
+    },
+    "icao": "KCHS",
+    "iata": "CHS",
+    "magnetic_north": -6.5,
+    "ctr_radius": 70,
+    "ctr_ceiling": 10000,
+    "initial_alt": 4000,
+    "position": ["N32d53m53.907", "W80d02m25.880","45ft"],
+    "rangeRings": {
+        "enabled": true,
+        "center": ["N32d53m53.907", "W80d02m25.880"],
+        "radius_nm": 5.0
+    },
+    "has_terrain": true,
+    "wind": {
+        "angle": 330,
+        "speed": 10
+    },
+    "arrivalRunway": "33",
+    "departureRunway": "33",
+    "defaultMaps": ["Base Map"],
+    "airspace": [
+        {
+            "floor": 0,
+            "ceiling": 100,
+            "airspace_class": "C",
+            "poly": [
```

Comment:
The airspace polygon should match the video map perfectly. You can use a tool like qGIS to help with that.
![image](https://user-images.githubusercontent.com/13125311/223114504-fc927c2a-2796-4cb3-950a-746d157a8d5b.png)